src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -509,25 +509,26 @@ public void createCaseConversionNote(String caseUUIDString, String stageUUIDStri
 
     private UUID deriveTeamUUID(String caseUUIDString, String stageTypeString, String allocationTeamString) {
         UUID teamUUID;
-        if (StringUtils.isEmpty(allocationTeamString)) {
+        if (!StringUtils.hasText(allocationTeamString)) {
             log.debug("Getting Team selection from Info Service for stage {} for case {}", stageTypeString, caseUUIDString);
             teamUUID = infoClient.getTeamForStageType(stageTypeString);
         } else {
             log.debug("Overriding Team selection with {} for stage {} for case {}", allocationTeamString, stageTypeString, caseUUIDString);
             teamUUID = UUID.fromString(allocationTeamString);
         }
-
+        log.info("Assigning teamUUID {} for case {} at stage {}",teamUUID, caseUUIDString, stageTypeString);
         return teamUUID;
 
     }
 
     private UUID deriveUserUUID(String caseUUIDString, String stageTypeString, String allocatedUserId) {
-        UUID userUUID = null;
-        if (!StringUtils.isEmpty(allocatedUserId)) {
-            log.debug("Assigning user {} to stage {} for case {}", allocatedUserId, stageTypeString, caseUUIDString);
-            userUUID = UUID.fromString(allocatedUserId);
+        if (StringUtils.hasText(allocatedUserId)) {
```

Comment:
Same here, worth logging out the allocatedUserId?

Reply:
I think the same is true here... if there is an `allocatedUserId` it's logged out in the next line. If there isn't, there's a `warn` level message explicitly stating it.